tic_tac_toe/src/board.py

```diff
@@ -151,10 +149,10 @@ def _check_diagonal(self, x_or_o):
         # Check of someone has won
         board_width = self.dim
         winner = None
+        values = set()
 
         for i in range(0, board_width):
-            values = set()
-            index = self.convert(i, i)
+            index = self.convert(i + 1, i + 1)
```

Comment:
This should be just `i`.  Also, you ignored the other diagonal.  write out the indices and figure out how to compute them.  I'll give you a hint, the indices are:
1. (0, 2)
2. (1, 1)
3. (2, 0)